flask_restalchemy/resources/utils.py

```diff
@@ -67,40 +68,33 @@ def get_field_serializer_or_none(serializer, field_name):
     return field.serializer
 
 
-def query_from_request(model, model_serializer, request, query=None):
+def apply_request_args_to_query(model, model_serializer, request, query):
```

Comment:
Since `query` is the object this function manipulates, it's more standard to set it as the first parameter.